frontend/src/metabase/reference/containers/ReferenceGettingStartedGuide.jsx

```diff
@@ -487,7 +487,7 @@ export default class ReferenceGettingStartedGuide extends Component {
                                     />
                                 </div>
                             ]}
-                            { Object.keys(metrics) > 0  && (
+                            { Object.keys(metrics).length > 0  && (
```

Comment:
Flow would catch it. I've been adding annotations for most new code and as I come across things, but it's a fair amount of work.